src/ble/link/advertising.rs

```diff
@@ -694,13 +694,13 @@ impl fmt::Debug for PduBuf {
 #[derive(Copy, Clone)]
 pub struct Header(u16);
 
-const TXADD_MASK: u16 = 0b00000000_01000000;
-const RXADD_MASK: u16 = 0b00000000_10000000;
+const TXADD_MASK: u16 = 0b0000000001000000;
+const RXADD_MASK: u16 = 0b0000000010000000;
```

Comment:
These underscores were there for a reason though - this is way harder to read now